data/alpha/register/statistical-geography-council-area-sct.yaml

```diff
@@ -0,0 +1,11 @@
+fields:
+- statistical-geography-council-area-sct
+- local-authority-sct
+- enacting-legislation
+- dissolving-legislation
+- start-date
+- end-date
+phase: alpha
+register: statistical-geography-council-area-sct
+registry: the-scottish-government
```

Comment:
Not a review comment, but this isn't this a great example of the problem with meaningful keys, because we couldn't remove the 'the' from the key, only the name :(

Reply:
rolling stones duncan, rolling stones

Reply:
I'd also like to double-check this, because on the discovery version of the register I believe the registry was initially 'The Scottish Government' before it was changed to 'Scottish Government'. Which is correct? https://statistical-geography-council-area-sct.discovery.openregister.org/

Reply:
it is how we list it in this file, which is what we match against in order for crests etc to work:

https://github.com/openregister/openregister-java/blob/master/src/main/resources/config/public-bodies.yaml

Reply:
I can't remember (or never knew) why we don't have a registry register.